sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Release History
+
+## 1.0.0-preview.1 (2020-01-09)
+
+- First preview release of Track 2 Text Analytics package.
+- Differences from previous Track 1 package:
```

Comment:
I don't believe we want to use the terms Track 1 and Track 2 in any official capacity in our docs